helix-core/src/test/java/org/apache/helix/integration/task/TestStopWorkflow.java

```diff
@@ -203,13 +202,20 @@ public void testResumeTaskForQuota() throws InterruptedException {
 
     _driver.start(workflowBuilder_1.build());
 
-    Thread.sleep(2000L); // Sleep until each task really is in progress
+    // Check the jobs are in progress. Each job has one task.
+    for (int i = 0; i < 30; i++) {
+      _driver.pollForJobState(workflowName_1, workflowName_1 + "_JOB" + i, TaskState.IN_PROGRESS);
+    }
```

Comment:
Nit: Could a job be in progress but its task not in RUNINNG state? Do you need to check for the task state?

Reply:
Yes. It can also be in INIT. I added the check got the tasks.